src/main/java/creativei/service/impl/FollowUpServiceImpl.java

```diff
@@ -35,30 +35,26 @@ public FollowUp updateFollowUp(FollowUp followUp) throws InvalidParamRequest {
     }
 
     private Boolean validateFollowUp(FollowUp followUp) {
```

Comment:
Add an info log here saying "Inside validateFollowUp method"

Reply:
logger added in commit 0d8af9c